### ISSUE TITLE
Metric Logging

### ISSUE THREAD

**Issue by kirilklein** [FGA-DIKU/EHR#186]
Combined metrics are not logged properly:
![Image](https://github.com/user-attachments/assets/c1761ad1-bf22-4ea7-9c15-1144f0d0434d)
Also these metrics are missing from the fold metrics so it's impossible to see the current learning rate

**Comment by kirilklein** [comment]
![Image](https://github.com/user-attachments/assets/d08b5721-e5ef-4c82-92ca-c3a19de1adda)
probably not an issue with folds, since it also occurs during pt. Seems like lr and train_loss has actual jumps. Could this be because of mixed precision? @mikkelfo

**Comment by kirilklein** [comment]
Can't reproduce.

**Comment by StephanLorenzen** [comment]
It might be a combination - for sure they are incorrectly logged outside the folds in finetune, and if there is more than a single fold, the graphs will be messed up. I will fix that.

**Comment by mikkelfo** [comment]
Unclear what exactly is going on... Shouldn't be mixed precision due to learning rate also being affected. Does it consistently happen after a few epochs, no matter if it's pretrain or fine-tune? @kirilklein @StephanLorenzen

**Comment by kirilklein** [comment]
> Unclear what exactly is going on... Shouldn't be mixed precision due to learning rate also being affected. Does it consistently happen after a few epochs, no matter if it's pretrain or fine-tune? [@kirilklein](https://github.com/kirilklein) [@StephanLorenzen](https://github.com/StephanLorenzen)

Yes

**Comment by mikkelfo** [comment]
@StephanLorenzen will prefix individual folders rather than using sub jobs for each fold
